lib/SIL/ValueOwnershipKindClassifier.cpp

```diff
@@ -132,6 +132,10 @@ CONSTANT_OWNERSHIP_INST(Unowned, UnownedToRef)
   }
 CONSTANT_OR_TRIVIAL_OWNERSHIP_INST(Guaranteed, StructExtract)
 CONSTANT_OR_TRIVIAL_OWNERSHIP_INST(Guaranteed, TupleExtract)
+// Because of CoW existentials, ownership cannot be transfered to the opened
+// value. Opening the value requires borrowing the existential and copying the
+// opened value.
```

Comment:
I think this comment is a bit misleading since the prose makes it sound like opening causes a copy. I took what you had here and combined it with your awesome text comment, what do you think?

```
OpenExistentialValue opens the boxed value inside an existential
CoW box. The semantics of an existential CoW box implies that we
can only consume the projected value inside the box if the box is
unique. Since we do not know in general if the box is unique
without additional work, in SIL we require opened archetypes to
be borrowed sub-objects of the parent CoW box.
```